src/source/tile.js

```diff
@@ -29,7 +31,13 @@ class Tile {
         this.sourceMaxZoom = sourceMaxZoom;
         this.buckets = {};
         this.expires = null;
-        this.cacheControl = null;
+        this.expired = false;
```

Comment:
On the theory that all time-based logic is horrible to edit/debug, could we rename `this.expires` to something like `this.expirationTime` and `this.expired` to `this.isExpired`?